DQM/L1TMonitor/python/L1TCSCTF_cfi.py

```diff
@@ -17,5 +17,6 @@
 #
 # Make changes for running in Run 2
 #
-from Configuration.StandardSequences.Eras import eras
-eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
+# This is not working.
+#from Configuration.StandardSequences.Eras import eras
+#eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
```

Comment:
> On Jul 7, 2015, at 1:54 PM, Ren-Jie Wang <notifications@github.com> wrote:
> 
> In DQM/L1TMonitor/python/L1TCSCTF_cfi.py:
> 
> > @@ -17,5 +17,6 @@
> >  #
> >  # Make changes for running in Run 2
> >  #
> > -from Configuration.StandardSequences.Eras import eras
> > -eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
> > +# This is not working.
> > +#from Configuration.StandardSequences.Eras import eras
> > +#eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
> 
> Yes, but this line does not work, I std::cout this flag in code, and it never went to "False”.

Right - did you read my comment?

"(which will eventually be replaced by the eras.run2…."

> 

> —
> Reply to this email directly or view it on GitHub.
>